src/test/isolation2/workfile_mgr_test.c

```diff
@@ -759,6 +759,9 @@ workfile_made_in_temp_tablespace(void)
 	if(0 != strncmp(bufFilePath, expectedPathPrefix, strlen(expectedPathPrefix)))
 		success = false;
 
+	/* FIXME: Should test this? */
+	(void) success;
```

Comment:
I think we should remove the unused variable or use it, but not make a compiler silent, adding useless lines.

Reply:
The check should be implemented in another way. The success variable is set to false, when bufFile == NULL, but if bufFile is NULL, then we get segfault on the next line `BufFileGetFilename(bufFile)`. You can create a new ticket for research instead of writing FIXME comment, because we don't find and read these comments usually. So I suggest to remove the variable in this patch.